setup.py

```diff
@@ -26,7 +26,7 @@
 release_status = "Development Status :: 5 - Production/Stable"
 dependencies = [
     "google-api-core[grpc] >= 1.14.0, < 2.0.0dev",
-    "google-cloud-core >= 1.0.3, < 2.0dev",
+    "google-cloud-core >= 1.4.0, < 2.0dev",
```

Comment:
```suggestion
    "google-cloud-core >= 1.4.1, < 2.0dev",
```

(I borked the dependencies in 1.4.0 😅 )